scripts/update-bindata.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script ensures that enable type directives are not merged to
+# the tree without corresponding fixture to ensure federation of the
+# target type will be tested.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT_DIR="$(cd "$(dirname "$0")/.." ; pwd)"
+
+function update-bindata(){
+  if command -v go-bindata > /dev/null; then
+    echo "go-bindata found."
+  else
+    echo "Downloading go-bindata"
+    go get -u github.com/jteeuwen/go-bindata/go-bindata/...
+  fi
+
+ "go-bindata \
+  -nocompress \
+  -nometadata \
+  -pkg 'common' \
+  -o ${ROOT_DIR}/bindata_generated.go \
```

Comment:
Would it make sense to output to `${ROOT_DIR}/test/common/bindata.go`, the actual location of the generated content, so that users can easily commit the modified changes if there are any?